rasa/core/training/story_reader/markdown_story_reader.py

```diff
@@ -257,6 +295,26 @@ def is_markdown_story_file(file_path: Union[Text, Path]) -> bool:
             )
             return False
 
+    @staticmethod
+    def is_markdown_test_stories_file(file_path: Union[Text, Path]) -> bool:
+        """Checks if a file contains test stories.
+
+        Args:
+            file_path: Path of the file which should be checked.
+
+        Returns:
+            `True` if it's a file containing test stories, otherwise `False`.
```

Comment:
what about if it starts with `DEFAULT_TEST_STORIES_FILE_PREFIX ` ?

Reply:
logic seems to be different from yaml

Reply:
yes that is intentional, the logic is the same as before - don't want to change this as it would break the existing logic